src/azure-cli-core/azure/cli/core/sdk/policies.py

```diff
@@ -96,3 +96,13 @@ def on_response(self, request, response):  # pylint: disable=unused-argument, no
                         _LOGGER.debug(response.http_response.text())
         except Exception as err:  # pylint: disable=broad-except
             _LOGGER.debug("Failed to log response: %s", repr(err))
+
+
+class RecordTelemetryUserAgentPolicy(UserAgentPolicy):
+    def __init__(self, base_user_agent=None, **kwargs):
+        super().__init__(base_user_agent=base_user_agent, **kwargs)
```

Comment:
No need to override `__init__` if it does nothing more than calling `super().__init__`.